test/unit/utilities/conversion.js

```diff
@@ -0,0 +1,313 @@
+    });
+  });
+
+  suite('p5.prototype.byte', function() {
+    test('should be a function', function() {
+      assert.ok(myp5.byte);
+      assert.typeOf(myp5.byte, 'function');
+    });
+
+    test('should return 127 for 127', function() {
+      result = myp5.byte(127);
+      assert.strictEqual(result, 127);
+    });
+
+    test('should return -128 for 128', function() {
+      result = myp5.byte(128);
+      assert.strictEqual(result, -128);
+    });
+
+    test('should return 23 for 23.4', function() {
+      result = myp5.byte(23.4);
+      assert.strictEqual(result, 23);
+    });
+
+    test('should return 1 for true', function() {
+      result = myp5.byte(true);
+      assert.strictEqual(result, 1);
+    });
+
+    test('should return 23 for "23.4"', function() {
+      result = myp5.byte('23.4');
+      assert.strictEqual(result, 23);
+    });
+
+    test('should return NaN for invalid string', function() {
+      result = myp5.byte('cat');
+      assert.isNaN(result);
+    });
+
+    test('should return array', function() {
+      result = myp5.byte([0, 255, '100']);
+      assert.typeOf(result, 'Array');
+      assert.deepEqual(result, [0, -1, 100]);
+    });
+  });
+
+  suite('p5.prototype.char', function() {
+    test('should be a function', function() {
+      assert.ok(myp5.char);
+      assert.typeOf(myp5.char, 'function');
+    });
+
+    test('should return the char representation of the number', function() {
+      result = myp5.char(65);
+      assert.typeOf(result, 'String');
+      assert.strictEqual(result, 'A');
+    });
+
+    test('should return the char representation of the string', function() {
+      result = myp5.char('65');
+      assert.strictEqual(result, 'A');
+    });
+
+    test('should return array', function() {
+      result = myp5.char([65, 66, '67']);
+      assert.typeOf(result, 'Array');
+      assert.deepEqual(result, ['A', 'B', 'C']);
+    });
+  });
+
+  suite('p5.prototype.unchar', function() {
+    test('should be a function', function() {
+      assert.ok(myp5.unchar);
+      assert.typeOf(myp5.unchar, 'function');
+    });
+
+    test('should return the integer representation of char', function() {
+      result = myp5.unchar('A');
+      assert.typeOf(result, 'Number');
+      assert.strictEqual(result, 65);
+    });
+
+    test('should return array of numbers', function() {
+      result = myp5.unchar(['A', 'B', 'C']);
+      assert.typeOf(result, 'Array');
+      assert.deepEqual(result, [65, 66, 67]);
+    });
+  });
+
+  suite('p5.prototype.hex', function() {
+    test('should be a function', function() {
+      assert.ok(myp5.hex);
+      assert.typeOf(myp5.hex, 'function');
+    });
+
+    test('should return the hex representation of the number', function() {
+      result = myp5.hex(65);
+      assert.typeOf(result, 'String');
+      assert.strictEqual(result, '00000041');
+    });
+
+    test('should return Infinity for Infinity', function() {
```

Comment:
This is failing due to unknown behaviour, if it should return Infinity or "INFINITY" or "Infinity"